### PR TITLE
Dynamic routing bug

### PR DESCRIPTION
**Pull request by ryanbethel**
There is a dynamic routing bug for javascript pages. This PR shows a failing test for the problem.

### CODE REVIEW

File: src/http/any-catchall/router.mjs [hunk 1]
```diff
@@ -142,6 +142,8 @@ export default async function api (options, req) {
     return fingerprintPaths(_html(str, ...values))
   }
 
+  console.log("in here")
+
```
Comment (brianleroux) [inline src/http/any-catchall/router.mjs:146]: remove! 👆

File: src/http/any-catchall/router.mjs [hunk 2]
```diff
@@ -170,6 +172,8 @@ export default async function api (options, req) {
       res.html = html`${head({ req, status, error, store })}${raw}`
     }
     else {
+      console.log("in router")
+      console.log({pagePath})
```
Comment (brianleroux) [inline src/http/any-catchall/router.mjs:176]: also!

**Comment by brianleroux** [comment]
looks good otherwise 👍

File: src/http/any-catchall/router.mjs (hunk 2 above)
Comment (ryanbethel) [reply src/http/any-catchall/router.mjs:176]: 🤦 there gone.

File: src/http/any-catchall/router.mjs (hunk 1 above)
Comment (ryanbethel) [reply src/http/any-catchall/router.mjs]: done

**Review by kristoferjoseph — APPROVED** [review]

**Pull request merged**